recipes/libusockets/conda_build_config.yaml

```diff
@@ -0,0 +1,12 @@
+# See https://github.com/conda-forge/nodejs-feedstock/blob/6e33145837f3d719fed66d020e848d1636e55e38/recipe/conda_build_config.yaml#L19
+pin_run_as_build:
+  # libuv is missing run_exports
+  libuv:
+    max_pin: x.x
+    min_pin: x.x.x
```

Comment:
What's the difference between pinning in conda_build_config and using the pin_compatible jinja expression?

Reply:
To be honest, I am not sure. I took this approach from the linked nodejs-feedstock, but I did not considered alternative approaches.